apps/prairielearn/src/migrations/20230531184500_assessment_access_policies.sql

```diff
@@ -0,0 +1,24 @@
+-- enum_assessment_access_policy_type
+CREATE TYPE enum_assessment_access_policy_type AS ENUM ('manual', 'extension_token');
+
+
+
+-- Create assessment_access_policies table
+CREATE TABLE IF NOT EXISTS assessment_access_policies (
+    assessment_id BIGINT REFERENCES assessments(id) NOT NULL,
+    student_uid TEXT REFERENCES users(uid) ON UPDATE CASCADE ON DELETE CASCADE ,
```

Comment:
In the assessment_access_policies table, user_id is not a field, and since the instructor would be adding overrides using the uid of a student I thought it would be better to reference uid

Reply:
We generally always want to resolve UIDs to `user_ids`, as even a `uid` can change (for instance, if a student changes their Net ID).